fusestore/store/src/tests/service.rs

```diff
@@ -51,6 +51,9 @@ pub struct StoreTestContext {
 pub fn new_test_context() -> StoreTestContext {
     let mut config = configs::Config::empty();
 
+    // On mac File::sync_all() takes 10 ms ~ 30 ms, 500 ms at worst, which very likely to fail a test.
```

Comment:
Can we check the OS if it's MACOS and set it?

Reply:
> Can we check the OS if it's MACOS and set it?

Good idea 🤔

Reply:
All Non-Enterprise deice which has no Enhanced Power Loss Data Protection cannot sync enough fast. I think it's not just MacOS issue

Reply:
@templexxx 
You are totally right, as always! :DDD
Any suggestions for building a stable durable storage system would be appreciated!